oaknational/Oak-Web-Application
# chore(remove-unit-quiz): remove unit quiz from unitlistitem component

## Remove unit quiz reference from UnitListItem Component

- removes reference to units quiz

## Issue(s)

Fixes #1319 

## How to test

1. Go to https://deploy-preview-1320--oak-web-application.netlify.thenational.academy/beta/teachers/key-stages/ks4/subjects/maths/units?tier=foundation
2. Click on _______
3. You should see no unit quiz displayed on UnitListItemCards

## Screenshots

How it used to look (delete if n/a):
<img width="1396" alt="Screenshot 2023-02-27 at 11 37 35" src="https://user-images.githubusercontent.com/91190841/221554224-c1e01371-60ab-492b-8b55-272c12e55f4c.png">



How it should now look:
<img width="1379" alt="Screenshot 2023-02-27 at 11 35 01" src="https://user-images.githubusercontent.com/91190841/221554067-aec86798-9789-4c24-90bd-9a9ee6943547.png">



## Checklist
